### ISSUE TITLE
Suggestion : List of helpful Debug channels for certain issues

### ISSUE THREAD

**Issue by sootysax** [Courseplay/courseplay#3410]
I'd like to see a list of certain debugs that you would want to have for common issues that we might come up with, so we can post a log the first timethat is helpful to you without you having to tell us to redo it with a certain debug enabled..

For instance:

Issues with refill triggers should have debugs x, y, and z.
Issues with tip triggers should have a, b and c
Issues with field work should have.. etc

especially for newbies who have never had to post a log before, having this as part of the 'how to generate a proper bug report' would be helpful

**Comment by ThomasGaertner** [comment]
yea that's a TODO for me

**Comment by pvaiko** [comment]
It is also in the tooltips of the debug channel buttons on the HUD.

**Comment by Hubcap-1** [comment]
I agree this would be very helpful to everyone. A Posted list to be found easily would be the best
